src/main/java/com/puppetlabs/jenkins/plugins/puppetgatling/PuppetGatlingPublisher.java

```diff
@@ -332,9 +339,9 @@ else if (key.equals("blockdevice_sda_size")){
             logger.println("[PuppetGatling] - OS Data saved.");
         }
 
-        FilePath facterDataPath = new FilePath(workspace, "puppet-gatling/" + simID + "/gatling_sim_data.csv");
-        String facterData = IOUtils.toString(facterDataPath.read(), "UTF-8");
-        simReport.setFacterData(facterData);
+        FilePath simConfigDataPath = new FilePath(workspace, "puppet-gatling/" + simID + "/gatling_sim_data.csv");
+        String simConfigDataString = IOUtils.toString(simConfigDataPath.read(), "UTF-8");
+        simReport.setSimConfigDataString(simConfigDataString);
```

Comment:
This is just a result of a rename of a variable/accessors whose original names did not make any sense at all.